x-pack/plugins/enterprise_search/common/ml_inference_pipeline/index.test.ts

```diff
@@ -79,10 +116,28 @@ describe('getSetProcessorForInferenceType lib function', () => {
         "Copy the predicted_value to 'dest' if the prediction_probability is greater than 0.5",
       field: destinationField,
       if: 'ml.inference.dest.prediction_probability > 0.5',
+      on_failure: [
```

Comment:
This seems like a reasonable solution. I wonder how interested we are in the these errors being on the document though. 

I also wonder if there would a good way to have the `if` clause check is `ml.inference.dest` exists on the document, so this processor could skip if its not found instead of failing.

Reply:
We _could_ do `ignore_failure: true`, but I'm not sure we want to do that.

Otherwise, ya maybe updating the `if` to start with `ctx.ml.inference.dest && ` would work?

Reply:
There are null checks now 👍